arkouda/dtypes.py

```diff
@@ -66,16 +69,17 @@ def __repr__(self) -> str: # type: ignore
 
 ARKOUDA_SUPPORTED_INTS = (int,np.int64)
```

Comment:
add uint64 here

Reply:
also below https://github.com/Bears-R-Us/arkouda/blob/293da309874bcd010b2918965fe569df16ad3724/arkouda/dtypes.py#L146-L149
needs to be modified to split the `uint` and `int`  interpretation apart.